tests/unit/createBirthdaySquad.test.ts

```diff
@@ -37,8 +37,8 @@ const constants = vi.hoisted(() => ({
   teamId: "T001",
   otherUserIds: ["U002", "U003", "U004", "U005", "U006"],
   conversationId: "CH001",
-  krisztaUserId: "KU001",
-  mateUserId: "MU001",
+  adminUserId: "KU001",
```

Comment:
The K stands for Kriszta, but thats just my ocd. 
```suggestion
  adminUserId: "AU001",
```

Reply:
Nice! Changed.